src/app/pages/dynamic-stepper/dynamic-stepper.component.html

```diff
@@ -0,0 +1,75 @@
+<mat-toolbar color="primary" class="app-bar">
+    <button *ngIf="isSmall" style="margin-left: -8px;" mat-icon-button (click)="toggleMenu()">
+        <mat-icon>menu</mat-icon>
+    </button>
+    <div class="mat-title">Dynamic Stepper</div>
+    <pxb-spacer></pxb-spacer>
+    <button
+        *ngIf="!procedureFinished"
+        mat-icon-button
+        matTooltip="Clear all steps"
+        (click)="onRemoveAll()"
+        data-cy="delete-all"
+    >
+        <mat-icon>delete</mat-icon>
+    </button>
+</mat-toolbar>
+<div *ngIf="!procedureFinished">
+    <mat-vertical-stepper [linear]="true" #stepper>
+        <mat-step *ngFor="let step of steps; let i = index" [completed]="step.completed">
+            <form>
+                <ng-template matStepLabel
+                    >{{ step.title || 'Choose an action' }}
+                    <mat-icon
+                        class="delete_icon"
+                        *ngIf="stepper.selectedIndex === i"
+                        data-cy="delete-step"
+                        (click)="removeStep(i)"
+                        matTooltip="Remove step"
+                        matTooltipPosition="right"
+                        >delete</mat-icon
+                    >
+                </ng-template>
+                <mat-radio-group class="radio-group" (change)="changeStepSelection($event, i)" data-cy="radiogroup">
+                    <mat-radio-button style="margin: 10px 0;" *ngFor="let option of stepOptions" [value]="option.value">
```

Comment:
Can we update all the margins/paddings in here that are using a multiple of 5 (old standard) to use multiples of 8 (material standard)?

Reply:
There might be an instance or two of this in the patterns that are already merged in too